src/main/java/de/lukweb/justmail/smtp/SmtpResponse.java

```diff
@@ -20,7 +20,7 @@
     ARGUMENT_ERROR(501, "Synatx error in parameters or arguments"),
     COMMAND_NOT_IMPLEMENTED(502, "SmtpCommand not implemented"),
     BAD_SEQUENCE(503, "Bad sequence of commands"),
-    ARGUMENT_NOT_IMPLEMENTED(504, "SmtpCommand parameter not implemented"),
```

Comment:
According to [Status Codes](https://www.iana.org/assignments/smtp-enhanced-status-codes/smtp-enhanced-status-codes.xhtml), 504 is for invalid command arguments, so i will change it to that